web/react/components/create_post.jsx

```diff
@@ -237,8 +243,8 @@ export default class CreatePost extends React.Component {
         Utils.setCaretPosition(React.findDOMNode(this.refs.textbox.refs.message), newText.length);
     }
     removePreview(id) {
-        let previews = this.state.previews;
-        let uploadsInProgress = this.state.uploadsInProgress;
+        const previews = $.extend(true, [], this.state.previews);
```

Comment:
Can that be used for arrays? I looked at the documentation quickly and it didn't mention arrays